WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListView.swift

```diff
@@ -148,14 +164,14 @@ struct CustomFieldsListView: View {
                 .listStyle(.plain)
             }
             .sheet(item: $viewModel.selectedCustomField) { customField in
-	            /// When editing a newly added and unsaved custom field (identified by it having nil `fieldId`), provide disallowed keys.
-	            let disallowedKeys = customField.fieldId == nil ? viewModel.disallowedKeysForCreation : []
+                /// When editing a newly added and unsaved custom field (identified by it having nil `fieldId`), provide disallowed keys.
+                let disallowedKeys = customField.fieldId == nil ? viewModel.disallowedKeysForCreation : []
 
-	            buildCustomFieldEditorView(customField: customField,
-	                                       disallowedKeys: disallowedKeys)
+                buildCustomFieldEditorView(customField: customField,
+                                           disallowedKeys: disallowedKeys)
             }
             .sheet(isPresented: $viewModel.isAddingNewField) {
-	            buildCustomFieldEditorView(customField: nil, disallowedKeys: viewModel.disallowedKeysForCreation)
+                buildCustomFieldEditorView(customField: nil, disallowedKeys: viewModel.disallowedKeysForCreation)
```

Comment:
Did not realize this was committed too. I think Xcode automatically formatted the spacing but there's no other code change here.